x/registration/remote_attestation/remote_attestation.go

```diff
@@ -135,6 +131,10 @@ func verifyCert(payload []byte) ([]byte, error) {
 
 	opts := x509.VerifyOptions{
 		Roots: roots,
+		// note: there's no way to not validate the time, and we don't want to write this code
+		// ourselves. We also can't just ignore the error message, since that means that the rest of
+		// the validation didn't happen (time is validated early on)
+		CurrentTime: time.Date(2023, 11, 04, 00, 00, 00, 00, time.UTC),
```

Comment:
What does this mean?

Reply:
Remember the 90 day certificate thing from a while back? It's the fix for that

Reply:
I merged a few older PRs here too, for good measure